src/com/facebook/buck/rules/macros/ClasspathMacroExpander.java

```diff
@@ -85,7 +85,7 @@ protected Arg expand(SourcePathResolver resolver, ClasspathMacro ignored, BuildR
   // The # characters that might be present in classpaths due to flavoring would be read as
   // comments. As a simple workaround, we quote the entire classpath.
   private static class ClassPathWriteToFileArg extends WriteToFileArg {
-    public ClassPathWriteToFileArg(BuildTarget target, String prefix, Arg delegate) {
```

Comment:
Changes in this file look unrelated, could you revert them?

Reply:
Intellij was showing suggestions that this is not needed, will revert back.